Automattic/wp-e2e-tests
# Wait for categoriesAndTagsSelector

From time to time we have failure like [this](https://circleci.com/gh/Automattic/wp-e2e-tests/24943) on master, with error message `New tag #Tag 1544195074439 not applied`. According to [screenshot](https://24943-57936731-gh.circle-artifacts.com/1/home/circleci/wp-e2e-tests/screenshots/FAILED-EN-DESKTOP-verify-tags-present-after-save-1544194543047.png), tag is present. 
With additional `waitTillPresentAndDisplayed()` test should be more reliable. 

To test:
Make sure that `Public Posts: Preview and Publish a Public Post @parallel @jetpack` is passing.
